docs/transactions/source/transaction-components/transaction-id.rst

```diff
@@ -23,11 +24,8 @@ Here are the steps to compute it.
       "metadata": metadata
     }
```

Comment:
Should the `id` field be added?

Reply:
Oh, good catch, I forgot about that other page! I'll go fix it now.